Dockerfile

```diff
@@ -35,10 +35,34 @@ RUN wget https://repo.anaconda.com/miniconda/Miniconda3-py310_23.1.0-1-Linux-x86
         | sha256sum --check \
     && bash Miniconda3-py310_23.1.0-1-Linux-x86_64.sh -b -p /opt/miniconda3 \
     && rm Miniconda3-py310_23.1.0-1-Linux-x86_64.sh
-
 ENV PATH=/opt/miniconda3/bin:/opt/miniconda3/condabin:$PATH
 ENV LD_LIBRARY_PATH=/usr/local/cuda/lib:/usr/local/cuda/lib64:$LD_LIBRARY_PATH
 
+# Install a few additional utilities via pip
```

Comment:
Can you replicate these changes in `Dockerfile.olmo`?

Reply:
done

Reply:
I dont see the `gpustat`, `yes | unminimize` etc changes?

Reply:
oh good catch, incomplete copy/paste. they should line up now